apps/sharry/docker-compose.yml

```
@@ -0,0 +1,53 @@
+services:
+  sharry:
+    depends_on:
+     - db
```

yamllint

[warning] wrong indentation: expected 6 but found 5

(indentation)